app/views/admin/projects.html.erb

```diff
@@ -73,52 +48,107 @@ See doc/COPYRIGHT.rdoc for more details.
           <label for='name'><%= Project.model_name.human %>:</label>
           <%= text_field_tag 'name', params[:name] %>
       </li>
+      <li class="simple-filters--filter">
+        <label for='' aria-visible="false" style="visibility: hidden"><%= l(:button_apply) %></label>
```

Comment:
I feel ashamed for that hack, but the underlying issue is that all simple-filters should be refactored to match the new visual.. *sigh*.

I'll add a big TODO / hack notice to that one, it needs to leave by next week.